readme.txt

```diff
@@ -124,7 +124,7 @@ Currently, the following add-ons are available for Event Tickets:
 
 = [4.9] TBD =
 
-
+* Tweak - Adjust rsvp duration picker wider to allow more of the date to be shown
```

Comment:
I'm not sure if we are adding individual change log entries now at the plugin level, any comments @bordoni ?